pkg/utils/net-attach-def.go

```diff
@@ -0,0 +1,257 @@
+// Copyright (c) 2019 Kubernetes Network Plumbing Working Group
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package utils
+
+import (
+	"encoding/json"
+	"fmt"
+	"regexp"
+	"strings"
+	//"k8s.io/client-go/rest"
```

Comment:
Please drop unused imports. There are more of them in this and other added modules.

Reply:
fixed!